index.html

```diff
@@ -3,115 +3,55 @@
-          </div>
-        </section>
-        <section id="skills" class="section-box" >
-          <ul class="section-flex">
-            <li class="list-item"><h3>Skills</h3></li>
-            <li class="list-item">keen perception</li>
-            <li class="list-item">eager to learn</li>
-            <li class="list-item">patient</li>
-            <li class="list-item">attention to detail</li>
-            <li class="list-item">great sense of responsibility</li>
-            <li class="list-item">teamplayer</li>
-          </ul>
-        </section>
-        <section id="interests" class="section-box">
-          <ul class="section-flex">
-            <li class="list-item"><h3>Interests</h3></li>
-            <li class="list-item">Scuba Diving</li>
-            <li class="list-item">Crossfit</li>
-            <li class="list-item">Trekking</li>
-            <li class="list-item">Travelling</li>
-            <li class="list-item">Cooking</li>
-            <li class="list-item">Sustainability</li>
-          </ul>
-       </section>
-      </div>
-      <div class="wrapper">
-        <section id="life" class="section-box" >
-            <h3>Biography</h3>
-            <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
-        </section>
-        <section class="section-box">
-          <div class= "section-cv-flex">
-            <div class="cv-column">
-              <h3>2020</h3>
-            </div>
-            <div class="cv-column">
-              <h4>Web Development Bootcamp</h4>
-              <p>neuefische GmbH, Germany</p>
-            </div>
-            <div class="cv-column">
-              <h3>2020 - 2021</h3>
-            </div>
-            <div class="cv-column">
-              <h4>Personal Assistant</h4>
-              <p>Various employers, Germany</p>
-            </div>
-            <div class="cv-column">
-              <h3>2013 - 2019</h3>
-            </div>
-            <div class="cv-column">
-              <h4>Dive Center Manager</h4>
-              <p>TGI Helengeli, Maldives</p>
-            </div>
-            <div class="cv-column">
-              <h3>2007 - 2010</h3>
-            </div>
-            <div class="cv-column">
-              <h4>B.Sc. in Biology</h4>
-              <p>University of Cologne, Germany<p>
-            </div>
-          </div>
-        </section>
+    <img class="img-head" src="https://source.unsplash.com/random/600x300" alt="">
```

Comment:
I am not sure, but i think the image needs to be in something, like html, header or main. However, i might be wrong :-)

Reply:
Yes, I am not sure either. Thank you!